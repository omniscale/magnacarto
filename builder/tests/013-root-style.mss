[zoom > 1] {
  /* Applies to all layers at zoom > 1 */
  polygon-gamma: 0.3;
  #world {
    polygon-fill: #323;
  }
  #lakes {
    polygon-fill: #144;
  }
}