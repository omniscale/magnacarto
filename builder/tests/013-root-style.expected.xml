<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="" filter-mode="first">
    <Rule>
      <!--Zoom{>=2}-->
      <MaxScaleDenominator>200000000</MaxScaleDenominator>
    </Rule>
  </Style>
  <Layer name="" srs="" status="off" maximum-scale-denominator="200000000">
    <StyleName></StyleName>
  </Layer>
</Map>